doc/nrf/libraries/bin/lwm2m_carrier/app_integration.rst

```diff
@@ -41,30 +41,45 @@ The :ref:`lwm2m_carrier` sample project configuration (:file:`nrf/samples/nrf916
 
 You can provide the initialization parameter :c:type:`lwm2m_carrier_config_t` to overwrite the carrier default settings with the Kconfig options described below:
 
-
 * :kconfig:`CONFIG_LWM2M_CARRIER_CERTIFICATION_MODE`:
 
   * This configuration specifies if the LwM2M carrier library will connect to the carrier's certification servers or production servers.
   * Expected to be set during self-testing part of the certification process.
-  * This configuration is ignored if :kconfig:`CONFIG_LWM2M_CARRIER_USE_CUSTOM_BOOTSTRAP_URI` is set.
+  * This configuration is ignored if :kconfig:`CONFIG_LWM2M_CARRIER_USE_CUSTOM_SERVER_URI` is set.
 
-* :kconfig:`CONFIG_LWM2M_CARRIER_USE_CUSTOM_BOOTSTRAP_URI`:
+* :kconfig:`CONFIG_LWM2M_CARRIER_USE_CUSTOM_SERVER_URI`:
 
-  * This configuration, together with :kconfig:`CONFIG_LWM2M_CARRIER_CUSTOM_BOOTSTRAP_URI` lets the LwM2M carrier library connect to a bootstrap server other than the normal carrier server, thereby enabling the generic mode.
+  * This configuration, together with :kconfig:`CONFIG_LWM2M_CARRIER_USE_CUSTOM_URI` lets the LwM2M carrier library connect to a custom server other than the normal carrier server, thereby enabling the generic mode.
```

Comment:
And this one perhaps CONFIG_LWM2M_CARRIER_CUSTOM_URI?

Reply:
Thanks